tap_azure_git/__init__.py

```diff
@@ -1106,7 +1106,8 @@ def get_all_pipeline_runs(schema, org, repo_path, pipeline, state, mdata, start_
             run = {
                 **raw_run,
                 '_sdc_repository': sdc_repository,
-                '_sdc_id': '{}/pipeline/{}/run/{}'.format(sdc_repository, pipeline['id'], raw_run['id'])
+                '_sdc_id': '{}/run/{}'.format(sdc_repository, raw_run['id']),
+                '_sdc_parent_id': '{}/pipeline/{}'.format(sdc_repository, pipeline['id'])
```

Comment:
this could be called _pipeline_id but i think leaving it generic makes more sense. it's easier to find the parent in six months if we stick with this naming convention. the same comment applies to all of the places that i used _sdc_parent_id.

Reply:
I don't have strong opinions here.  `_sdc_parent_id` feels good to me.